packages/app-store/make/README.md

```diff
@@ -0,0 +1,8 @@
+# Setting up Make Integration
```

Comment:
@aar2dee2 I don't see the instructions on how to create the app on make.com.
We have something similar for zapier https://github.com/aar2dee2/cal.com/blob/6bfabe41e5073ff3a9e38d577ef6d72be2d6eca8/packages/app-store/zapier/README.md#L8

Reply:
I  just updated the readme. Request @CarinaWolli to look at point 5 - how did add the app to scenarios once you had the invite link?
[`5. Search for `Cal.com` in the apps list and select from the list of triggers - Booking Created, Booking Deleted, Booking Rescheduled, Meeting Ended`](https://github.com/calcom/cal.com/pull/8897/files#diff-01baf41f650a10552d5968a62b68c6d64b919b57c792ce7b9b4a55c36211835aR8)